### PR TITLE
refs task #28875 repositories for admin controller

### PR DESCRIPTION
**Pull request by vuchuc789**
## Related Tickets:
- [#28875](https://edu-redmine.sun-asterisk.vn/issues/28875)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by vuchuc789** [comment]
ready

File: app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -2,31 +2,38 @@
 
 namespace App\Http\Controllers;
 
-use App\Models\Role;
 use App\Models\Tag;
```
Comment (huyenttm-0901) [inline app/Http/Controllers/AdminController.php]: dùng repo thì bỏ model đi em

Comment (vuchuc789) [reply app/Http/Controllers/AdminController.php]: Em đã sửa ạ.

**Comment by vuchuc789** [comment]
ready

**Pull request merged**